src/main/java/org/terasology/launcher/packages/GamePackageType.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.launcher.packages;
+
+/**
+ * Enum for types of game package available now.
+ */
+public enum GamePackageType {
+    STABLE("TerasologyStable"),
+    UNSTABLE("Terasology"),
+    OMEGA_STABLE("DistroOmegaRelease"),
+    OMEGA_UNSTABLE("DistroOmega");
+
+    // TODO: Remove this enum when 3rd-party package types are supported
```

Comment:
Is the intent for the whole enum to go away in favor of JSON-based config files of some sort? I might suggest trying to work that out with even current state only supporting Jenkins.

Something like a `module.txt` but for packages, with description, url, provider type, etc. Might be something like that one piece to follow, even if there's no implementation (a design/convention from a different package manager that could be followed?)